lib/modules/notifications/notifications.dart

```diff
@@ -211,18 +213,48 @@ Future<void> scheduleNotificationsInner(
         prayerName = prayersTranslation[e.key] ?? "";
       }
 
-      await _flutterLocalNotificationsPlugin.zonedSchedule(
-        id,
-        t.prayerTimeNotificationTitle(prayerName),
-        t.prayerTimeNotificationContent(prayerName),
-        prayerTime,
-        athanPlatformChannelDetails,
-        androidScheduleMode: AndroidScheduleMode.exactAllowWhileIdle,
-        uiLocalNotificationDateInterpretation:
-            UILocalNotificationDateInterpretation.absoluteTime,
-      );
-      counter++;
-      id++;
+      // Schedule PreAthan notification
+      var preAthanSettings = prayerNotificationSettings.preAthanSettings;
+      if (preAthanSettings.enabled && counter < maxIOSNotification) {
```

Comment:
maybe check the counter only on ios platforms, android can surpass that limit

Reply:
The question is what is the limit?
Also, the workmanger should handle this on android, so not really sure about this.

Reply:
I meant it will reschedule in the background

Reply:
ok, np